doc/oemof_solph.rst

```diff
@@ -46,17 +46,16 @@ Now you can start to add the components of the network.
 Add your components to the energy system
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-If you have defined an instance of the EnergySystem class all components you define will automatically be added to your EnergySystem.
+After defining an instance of the EnergySystem class you have to add all components you define in the following to your EnergySystem.
 
 Basically, there are four types of Nodes and every node has to be connected with one or more buses. The connection between a component and a bus is the flow.
 
  * Sink (one input, no output)
  * Source (one output, no input)
- * LinearTransformer (one input, n outputs)
+ * LinearTransformer (n inputs, n outputs)
  * Storage (one input, one output)
```

Comment:
The storage is not a network class.

Reply:
Sorry, I did not noticed that we are already in the solph documentation.

In solph we do have three groups:

1. Components directly inherited from the network class (pure graph concept). They can be found in the network module and the blocks in the blocks module (We should change that because it is different to anything else).
2. Proofed solph components which can be found in the components module.
3. Experimental or very specific stuff in the custom module.

Reply:
Shall I add your explanation about the three groups to the docu?

Reply:
Why not, it helps to understand the structure.